helm/reana/README.md

```diff
@@ -62,6 +62,8 @@ This Helm automatically prefixes all names using the release name to avoid colli
 | `kubernetes_jobs_memory_limit`                           | Maximum default memory limit for user job containers. Exceeding this limit will terminate the container. Please see the following URL for possible values https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory. | 4Gi |
 | `kubernetes_jobs_max_user_memory_limit`                  | Maximum custom memory limit that users can assign to their job containers via `kubernetes_memory_limit` hint in `reana.yaml`. Exceeding this limit will terminate the container. Please see the following URL for possible values https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory. | None |
 | `node_label_infrastructure`                              | Define the label which identifies the nodes where the infrastructure pods should run, e.g. `reana.io/system=infrastructure`. If you are setting this configuration variable, please also set `.traefik.nodeSelector.reana.io/system=infrastructure` so the ingress controller is also deployed on the infrastructure node. By default infrastructure pods can be scheduled to any available node in the cluster. | None |
+| `node_label_infrastructuremq`                            | Define the label which identifies the node where message broker pod should run, e.g. `reana.io/system=infrastructuremq`. By default runtime workflow pods can be scheduled to any available node in the cluster. | None |
```

Comment:
I guess here we should mention "infrastructure pods" instead of "workflow pods"?

```suggestion
| `node_label_infrastructuremq`                            | Define the label which identifies the node where message broker pod should run, e.g. `reana.io/system=infrastructuremq`. By default infrastructure workflow pods can be scheduled to any available node in the cluster. | None |
```

Suggestion: What do you think about just calling the labels `node_label_mq` (`reana.io/system=mq`) and `node_label_db` (`reana.io/system=db`)? Is there any particular reason why we prepend "infrastructure"?